### PR TITLE
added info on Ads animations

### PR DESCRIPTION
**Pull request by CrystalOnScript**
Resolves issue [1075](https://github.com/ampproject/amp.dev/issues/1075)

@sebastianbenz and @matthiasrohmer - I couldn't get the filters didn't work on my local build, any idea why?

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md [hunk 1]
```diff
@@ -7,17 +7,37 @@ formats:
...
-Triggering CSS animations [relies on adding and removing classes](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations), done via JavaScript. You can achieve the same behavior on AMP pages by using the [`toggleClass`](/content/amp-dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.md#-all-elements) action. With `toggleClass`, AMP pages can add, remove, and toggle CSS classes like non-AMP pages.
+CSS animations enable web elements to transition from one CSS style configuration to another. The browser can start defined animations on load, but event triggered CSS animations [rely on adding and removing classes](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations). AMP supports both animation types.
+
+Use CSS when you have a smaller, contained animation that doesn't need to be precisely timed.
 
 ## Defining CSS and keyframes
 
 You can define CSS in AMP in the following ways:
 
+[filter="websites, stories"]
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md]: syntax error, needs to be: `[filter formats="websites, stories"]`

http://localhost:8080/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting/?format=websites#filtering-sections

Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md]: omg lol

**Comment by CrystalOnScript** [comment]
Screen Shot 2019-12-20 at 1 41 49 PM
So I've fixed the filtering, but it still seems to have having the same [issue as I filed here](https://github.com/ampproject/amp.dev/issues/3293)

This is really going to be an issue when the [email filtering is merged](https://github.com/ampproject/amphtml/pull/26107)

cc @matthiasrohmer

**Comment by sebastianbenz** [comment]
`s/format=/formats=` 😀

**Pull request merged**